### ISSUE TITLE
generate new pangenome database from single bin

### ISSUE THREAD

**Issue by sirmicrobe** [merenlab/anvio#1142]
Hello!

Is there a way of generating a new pangenome database by extracting a single bin? The practical application for this would be to look at something like the accessory genome in more detail.

I think what I want to do is basically the same as what you do with the E. faecalis bin in the Infant Gut tutorial, but with a PAN.db instead of a PROFILE.db. The example that Meren came up with was something like this: 

```
anvi-split-pan -g GENOMES.db \
               -p PAN.db \
               --collection COLLECTION \
               --gc-bins "GC_bin_01,GC_bin_02,..." \
               -o SPLIT-PAN.db
```

Thank you!

**Comment by ozcan** [comment]
@meren, do you think as an alternative to `anvi-split` approach, is it possible to accept pangenome bin as an internal genome input for `anvi-pangenome`?

**Comment by meren** [comment]
Thanks for the report, @sirmicrobe!

I think that would be quite an elegant way to resolve this, @ozcan. I am not sure about its feasibility though since it's been forever since the last time I looked at those parts of the code :)

**Comment by meren** [comment]
Hi @sirmicrobe, we resolved this issue in #1143 if you take a look at [this example](https://github.com/merenlab/anvio/issues/1143#issuecomment-487211317), you can see that it is very similar to what you suggested. It is currently available in anv'o `v5.5`.


Best wishes,
